Modules/Microsoft365DSC/DSCResources/MSFT_TeamsCallQueue/MSFT_TeamsCallQueue.psm1

```diff
@@ -52,10 +52,44 @@ function Get-TargetResource
         [System.String]
         $TimeoutAction,
 
+        [Parameter()]
+        [ValidateSet("Queue","Forward","Disconnect","Forward","Voicemail","SharedVoicemail")]
+        [System.String]
+        $NoAgentAction,
```

Comment:
The schema shows 22 new parameters, but the Get/Set/Test methods only show 10 new parameters. Please make sure the parameter in the schema matches the parameters in the *-TargetResource functions.